Sample Applications/WPFGallery/MainWindow.xaml.cs

```diff
@@ -244,4 +228,38 @@ private void ControlsList_Loaded(object sender, RoutedEventArgs e)
         }
     }
 
+    private void SelectedItemChanged(TreeViewItem? tvi)
+    {
+        ControlsList_SelectedItemChanged();
+        if (tvi != null)
```

Comment:
Can we do a nullable enable use those expressions, rather than doing a null check this way?

Reply:
Was getting some errors due to nullable expressions, hence used this way right now.